hedera-node/src/main/java/com/hedera/services/state/merkle/MerkleUniqueToken.java

```diff
@@ -64,6 +64,10 @@ public MerkleUniqueToken() {
 		/* No-op. */
 	}
 
+	public MerkleUniqueToken(EntityId owner) {
```

Comment:
Why do we need this new constructor? Can we reuse the already existing one like the below?

`public MerkleUniqueToken( EntityId owner, byte[] metadata, RichInstant creationTime )`

Reply:
I thought it might look more convenient that way in the code.